scripts/image_vars.src

```diff
@@ -49,7 +49,7 @@ imageTagExists(){
 export postgres_GRADLE_IMG="va/abd_vro-postgres"
 export postgres_IMG="vro-postgres"
 
-export rabbitmq_GRADLE_IMG="va/abd_vro-rabbitmq"
+export rabbitmq_GRADLE_IMG="va/abd_rabbitmq"
```

Comment:
to double-check: is this change in scope of this PR?

Reply:
Nope. Took it out